src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -416,3 +429,139 @@ def run_job(self, job: LandingJob) -> bool:
             self.phab_trigger_repo_update(repo.phab_identifier)
 
         return True
+
+    def read_lando_config(
```

Comment:
These should probably go into a different module / class / mixin (maybe `lando.scm.utils`)